benchmark/BackendBench/eval.py

```diff
@@ -0,0 +1,547 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Naming nit since it's one file: `benchmark/backend_bench.py` after moving setup (see comment)